sphinx/themes/basic/static/doctools.js

```diff
@@ -252,6 +253,31 @@ var Documentation = {
     });
     var url = parts.join('/');
     return path.substring(url.lastIndexOf('/') + 1, path.length - 1);
+  },
+
+  /**
+   * init onKeyListernes (for navigation)
+   */
+  initOnKeyListeners: function() {
+    $(document).keyup(function(event) {
+      var activeElementType = $(document.activeElement).prop('tagName');
+      if (activeElementType !== 'TEXTAREA' && activeElementType !== 'INPUT') { // don't navigate when in search box or textarea
+        switch (event.keyCode) {
+          case 37: // left
+            var prevHref = $('link[rel="prev"]').prop('href');
+            if (prevHref) {
+              window.location.href = prevHref;
+            }
+            return false;
```

Comment:
Why is the `return` statement outside of the `if` clause?  Do we really want to stop propagation when we haven't done anything?